src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -389,7 +409,9 @@ private MemberSignatureComparer(
             RefKindCompareMode refKindCompareMode,
             bool considerArity = true,
             bool considerDefaultValues = false,
-            TypeCompareKind typeComparison = TypeCompareKind.IgnoreDynamic | TypeCompareKind.IgnoreNativeIntegers)
+            TypeCompareKind typeComparison = TypeCompareKind.IgnoreDynamic | TypeCompareKind.IgnoreNativeIntegers,
+            bool considerParameterNames = false,
```

Comment:
>considerParameterNames [](http://example.com/codeflow?start=17&length=22)

It is not obvious from this name, in fact it is unexpected, that this value has any impact on type parameter comparison